src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryRegEx.cs

```diff
@@ -0,0 +1,277 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Texl.Builtins;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Functions
+{
+    internal static partial class Library
+    {
+        // https://learn.microsoft.com/en-us/power-platform/power-fx/reference/function-ismatch        
+
+        public const string FULLMATCH = "FullMatch";
+        public const string STARTMATCH = "StartMatch";
+        public const string SUBMATCHES = "SubMatches";
+
+        private const string DefaultIsMatchOptions = "^c$";
+        private const string DefaultMatchOptions = "c";
+        private const string DefaultMatchAllOptions = "c";
+
+        /// <summary>
+        /// Enable [Is]Match[All] functions.
```

Comment:
>Enable [Is]Match[All] functions [](http://example.com/codeflow?start=12&length=31)

Suggestion: "Creates instances of the [Is]Match[All] functions and returns them so they can be added to the runtime."